python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -174,6 +196,14 @@ export default function Editor() {
     return await ufetch.get(ROUTE_TABLE.SERVER_STATUS);
   }, []);
 
+  const logEvent = useCallback((event: LogEvent, data?: LogEventData) => {
+    try {
+      datadogLogs.logger.info(event, data);
+    } catch (e) {
+      // Ignore logger errors for now
+    }
```

Comment:
Does this get handled automatically if we remove try-catch by `forwardErrorsToLogs` field?

Reply:
Two seperate things.

This try catch ignores any errors by datadog logging module.

`forwardErrorsToLogs` forwards any `console.warn` to logs